libvast/src/system/connect_to_node.cpp

```diff
@@ -13,14 +13,15 @@
 
 #include "vast/system/connect_to_node.hpp"
 
+#include "vast/config.hpp"
```

Comment:
It is still reasonable to have this include, since the macro constants declared there are used in this file directly. Also, if `#include "vast.config.hpp"` gets removed  from `fwd.hpp` at some point, we'd have to add it back here, and probably a lot of other places.

Reply:
`fwd.hpp` directly depends on `config.hpp`, so I doubt this'll ever happen. But this wasn't a change request anyways, more just a comment to let you know.